helm-charts/splunk-otel-collector/values.yaml

```diff
@@ -1111,6 +1111,11 @@ gateway:
   config: {}
 
 ################################################################################
+# Notice: Network Explorer related features should be considered to have an
+# ALPHA maturity level and be experimental. There may be breaking changes or
+# Network Explorer features may be replaced entirely with a better alternative
+# in the future.
+#
```

Comment:
This is k8s service section, not Network explorer

Reply:
I thought this section introduction was for the network explorer and more. Looks like the network explorer doesn't have a section introduction. Fixing.

Reply:
Addressed in https://github.com/signalfx/splunk-otel-collector-chart/pull/976